projects/RabbitMQ.Client/client/api/IBasicConsumer.cs

```diff
@@ -98,10 +95,10 @@ public interface IBasicConsumer
             ReadOnlyMemory<byte> body);
 
         /// <summary>
-        ///  Called when the model shuts down.
+        ///  Called when the channel shuts down.
         ///  </summary>
-        ///  <param name="model"> Common AMQP model.</param>
-        /// <param name="reason"> Information about the reason why a particular model, session, or connection was destroyed.</param>
+        ///  <param name="model">The channel.</param>
+        /// <param name="reason"> Information about the reason why a particular channel, session, or connection was destroyed.</param>
         void HandleModelShutdown(object model, ShutdownEventArgs reason);
```

Comment:
Will channel become mostly async? If yes we should probably be async all the way